presto-docs/src/main/sphinx/release/release-0.167-t.rst

```diff
@@ -0,0 +1,57 @@
+=================
+Release 0.167-t
+=================
+
+Presto 0.167-t is equivalent to Presto release 0.167, with some additional features and patches.
+
+**TIMESTAMP limitations**
+
+Presto supports a granularity of milliseconds for the ``TIMESTAMP`` datatype, while Hive
+supports microseconds.
+
+**TIMESTAMP semantic changes**
+
+Teradata distribution of Presto does introduce fix of semantic of ``TIMESTAMP`` type.
+
+Previously ``TIMESTAMP`` was a type describing instance of time in Presto session time zone.
+Currently Presto treats this - as specified SQL standard - as set of following fields representing wall time:
+
+ * ``YEAR OF ERA``
+ * ``MONTH OF YEAR``
+ * ``DAY OF MONTH``
+ * ``HOUR OF DAY``
+ * ``MINUTE OF HOUR``
+ * ``SECOND OF MINUTE`` - as decimal with precision 3
+
+For that reason ``TIMESTAMP`` value is not linked with session time zone in any way until time zone is needed explicitly.
+Those scenarios include casting to ``TIMESTAMP WITH TIME ZONE`` (and ``TIME WITH TIME ZONE``). In those scenarios
+time zone offset of session time zone and session time is applied as specified in SQL standard.
+
+For various compatibility reasons, when casting from data time type without time zone to one with time zone, fixed time
```

Comment:
For various compatibility reasons, when casting from a data type without a time zone to one with a time zone, a fixed time zone is used as opposed to the named on that may be set for the session.